src/instrument/tests/conftest.py

```diff
@@ -0,0 +1,31 @@
+"""
+Pytest fixtures for instrument tests.
+
+This module provides fixtures for initializing the RunEngine with devices,
+allowing tests to operate with device-dependent configurations without relying
+on the production startup logic.
+
+Fixtures:
+    runengine_with_devices: An initialized RunEngine with devices configured.
+"""
+
+from typing import Any
+
+import pytest
+
+from ..startup import RE
+from ..startup import make_devices
+
+
+@pytest.fixture(scope="session")
+def runengine_with_devices() -> Any:
+    """
+    Initialize the RunEngine with devices for testing.
+
+    This fixture calls RE with the output of make_devices() to mimic
+    the behavior previously performed in the startup module.
+
+    Returns:
+        Any: An instance of the RunEngine with devices configured.
+    """
+    return RE(make_devices())
```

Comment:
Make it into two lines:

```suggestion
    RE(make_devices())
    return RE
```